frontend/src/pages/organization/SecretManagerOverviewPage/components/AllProjectView.tsx

```diff
@@ -0,0 +1,286 @@
+import { useState } from "react";
```

Comment:
we should also update the text here right?
<img width="1512" alt="image" src="https://github.com/user-attachments/assets/ff7d5300-87e3-418f-a50c-ac7de5411743" />